--- ./zshrc.sh
+++ ./zshrc.sh
@@ -48,7 +48,7 @@ function update_current_git_vars() {
         _GIT_STATUS=`python ${gitstatus} 2>/dev/null`
     fi
     if [[ "$GIT_PROMPT_EXECUTABLE" == "haskell" ]]; then
-        _GIT_STATUS=`git status --porcelain --branch &> /dev/null | $__GIT_PROMPT_DIR/src/.bin/gitstatus`
+        _GIT_STATUS=`git status --porcelain --branch &> /dev/null | gitstatus`
     fi
      __CURRENT_GIT_STATUS=("${(@s: :)_GIT_STATUS}")
 	GIT_BRANCH=$__CURRENT_GIT_STATUS[1]
